internal/config/config.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"github.com/vulpemventures/go-elements/network"
+	"time"
+)
+
+const (
+	// NeutrinoDUrlKey is the key for the neutrino url
+	NeutrinoDUrlKey = "NEUTRINOD_URL"
+	// ExplorerUrlKey is the URL of the Liquid network
+	ExplorerUrlKey = "EXPLORER_URL"
+	//PeerUrlKey is the URL of the peer node
+	PeerUrlKey = "PEER_URL"
+	// NetworkKey is the network to use. Either liquid, testnet or regtest
+	NetworkKey = "NETWORK"
+	// LogLevelKey are the different logging levels. For reference on the values https://godoc.org/github.com/sirupsen/logrus#Level
+	LogLevelKey = "LOG_LEVEL"
+)
+
+var (
+	vip *viper.Viper
+)
+
+func LoadConfig() error {
+	vip = viper.New()
+	vip.SetEnvPrefix("NEUTRINO")
+	vip.AutomaticEnv()
+
+	vip.SetDefault(NeutrinoDUrlKey, "localhost:8080")
```

Comment:
8080 is usually used by frontends

```suggestion
	vip.SetDefault(NeutrinoDUrlKey, "localhost:8000")
```

Reply:
Done